### DISCUSSION TITLE
Scramble visualisation

### DISCUSSION THREAD

**Discussion by JupiLogy** [kash/cubedesk#106] [Ideas]
One person in the discord requested visualisation for scrambles on pyraminx and some other puzzles that currently have no scramble preview.

I also noticed cubedesk uses scrambow to get scrambles.

Scramby is a fork of the same original project that scrambow was also forked from, but scramby has a 'draw scramble' function.

So, I'm thinking we can take advantage of the code in scramby for the drawing scrambles, and then put it into the cubedesk "style"? I'm happy to work on making the scramble drawing backend but as I said in my introduction post, front end and therefore the styling part of it wouldn't be easy for me.

Anyone want to do the style bit if I get the scramble drawing part working?

Is this a good idea @kash ?

**Comment by kash** [comment]
@JupiLogy Hey! Absolutely, this is one of the most request things by the community. Would be awesome if you'd like to work on this! There are instructions on getting the dev environment set up in the Wiki but please let me know if you get stuck. Can also help if you need guidance on what files to modify.
